tests/test_pyxis.py

```diff
@@ -0,0 +1,426 @@
+                        "published": False
+                    },
+                    {
+                        "manifest_list_digest": "sha256:1112",
+                        "published": True
+                    }
+                ]
+            },
+            {
+                "brew": {
+                    "build": "s2i-1-2",
+                    "completion_date": "2020-08-12T11:31:39+00:00",
+                    "nvra": "s2i-1-2.s390x",
+                    "package": "s2i-core-container"
+                },
+                "repositories": [
+                    {
+                        "manifest_list_digest": "sha256:2222",
+                        "published": True
+                    }
+                ]
+            },
+            {
+                "brew": {
+                    "build": "s2i-1-2",
+                    "completion_date": "2020-08-12T11:31:39+00:00",
+                    "nvra": "s2i-1-2.amd64",
+                    "package": "s2i-core-container"
+                },
+                "repositories": [
+                    {
+                        "manifest_list_digest": "sha256:3333",
+                        "published": True
+                    }
+                ]
+            },
+            {
+                "brew": {
+                    "build": "s2i-1-2",
+                    "completion_date": "2020-08-12T11:31:39+00:00",
+                    "nvra": "s2i-1-2.arm64",
+                    "package": "s2i-core-container"
+                },
+                "repositories": [
+                    {
+                        "manifest_list_digest": "sha256:4444",
+                        "published": True
+                    }
+                ]
+            }
+        ]
+
+    def tearDown(self):
+        super().tearDown()
+        self.patcher.unpatch_all()
+        self.koji_read_config_patcher.stop()
+
+    @staticmethod
+    def copy_call_args(mock):
+        """
+        Copy args of Mock to another Mock so we can check call args if we call
+        mock with mutable args and change it between calls
+        """
+        new_mock = Mock()
+
+        def side_effect(*args, **kwargs):
+            args = deepcopy(args)
+            kwargs = deepcopy(kwargs)
+            return new_mock(*args, **kwargs)
+        mock.side_effect = side_effect
+        return new_mock
+
+    @patch('freshmaker.pyxis.HTTPKerberosAuth')
+    @patch('freshmaker.pyxis.requests.get')
+    def test_make_request(self, get, auth):
+        get.return_value = self.response
+        test_params = {'key1': 'val1'}
+        self.px._make_request('test', test_params)
+
+        get_url = self.fake_server_url + 'v1/test'
+        self.response.json.assert_called_once()
+        test_params['page_size'] = "100"
+        get.assert_called_once_with(get_url, params=test_params, auth=auth(),
+                                    timeout=conf.net_timeout)
+
+    @patch('freshmaker.pyxis.HTTPKerberosAuth')
```

Comment:
My suggestion is to mock `_make_request` since it's already tested in `test_make_request` and you don't need to test it again. This makes the tests more independent of each other so in the event that the implementation of `_make_request` changes, you don't need to change `test_pagination`.